README.md

```diff
@@ -207,3 +214,7 @@ secure the connection to the SMTP server
 
 * `password`: password to be used to authenticate to the SMTP server (only
 used if `user` is specified)
+
+* 'headers': dictionary of string keys and string values to be added as
+custom headers; if 'From', 'To', 'Cc', or 'Bcc' are included here they
+will override the corresponding configuration keys above
```

Comment:
Or maybe we should disallow those accounted for in other keys?

Reply:
Yes, that's probably safer. Will add that.